app/lang/en.json

```diff
@@ -197,6 +197,14 @@
     "certificateProviderDobHint": "We need your certificate provider's date of birth to check they are over 18.",
     "mobile": "Mobile number",
 
+    "howWouldCertificateProviderPreferToCarryOutTheirRole": "How would {{.FirstNames}} prefer to carry out their role?",
+    "selectOneOption": "Select one option",
+    "onlineAndByEmail": "Online and by email",
+    "onlineAndByEmailHint": "We can process your LPA application quicker if your certificate provider chooses this option.",
+    "usingPaperForms": "Using paper forms",
+    "certificateProvidersEmail": "Certificate provider's email address",
+    "certificateProviderAddress": "{{.FirstNames}}'s address",
+
```

Comment:
Looks like we need `selectHowWouldCertificateProviderPreferToCarryOutTheirRole` added as a translation.